apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/keywords.kt

```diff
@@ -83,3 +83,9 @@ object only : Keyword
  */
 object order : Keyword
 
+/**
+ * It can be used for a parameter less function so that it has one parameter and thus can be used as infix function.
```

Comment:
you removed too much, please re-add the first sentence again:
```suggestion
 * Represents the pseudo keyword `success` as in [toBe] `success.
 * It can be used for a parameter less function so that it has one parameter and thus can be used as infix function.
```

Reply:
Looks like I've missed it when I was moving it to old interface. Fixed